src/components/TimeField/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { TimeInput } from '@scaleway/ui'
+import React, { ComponentProps } from 'react'
+import { useField } from 'react-final-form'
+import pickValidators from '../../helpers/pickValidators'
+import useValidation from '../../hooks/useValidation'
+
+type TimeFieldProps = {
+  name: string
+  initialValue?: Date
+  schedule?: ComponentProps<typeof TimeInput>['schedule']
+  placeholder?: string
+  required?: boolean
+  disabled?: boolean
+  validate?: (value: Date) => boolean
+}
+
+const parseTime = (date?: Date | string): { label: string; value: string } => {
+  const timeStr =
+    date && typeof date !== 'string'
+      ? date.toLocaleTimeString().slice(0, -3)
+      : ''
+
+  return {
+    label: timeStr,
+    value: timeStr,
+  }
+}
+
+const TimeField = ({
+  required,
+  name,
+  schedule,
+  placeholder,
+  disabled,
+  initialValue,
+  validate,
+}: TimeFieldProps) => {
+  const validateFn = useValidation<Date>({
+    validate,
+    validators: pickValidators<Date>({
+      required,
+    }),
+  })
+
+  const { input, meta } = useField<Date>(name, {
+    initialValue,
+    validate: validateFn,
+  })
+
+  return (
+    <TimeInput
+      placeholder={placeholder}
+      schedule={schedule}
+      required={required}
+      value={parseTime(input.value)}
```

Comment:
can we use parse on the useField instead ? 

https://final-form.org/docs/react-final-form/types/FieldProps#parse

Reply:
After trying it out, I figured that's not possible because on the input level I want the value to be a `Date` (to make it work in symbiosis with `DateField`) but on the `TimeInput` component level, I want the value to be `{ value: string, label: string }`